client-js/app/elements/app-link.html

```diff
@@ -1,5 +1,7 @@
 <script>
 (function () {
+  const MOUSE_LEFT_BUTTON = 0;
```

Comment:
Probably shouldn't use const here since this is just inline javascript, so we're not transpiling this to ES5 like we do with the typescript and I'm not sure about browser support of const. Of course, we can and probably should define this element in typescript like the other elements, in which case bringing const back would be fine.